data/games/under_night_in_birth_exe_late_st/images/nanase/index.js

```diff
@@ -1,6 +1,126 @@
 export default [
-  // {
-  //   "imageId": "5A",
-  //   "image": require('./5A.png')
-  // }
+  {
+    "imageId": "Halte_Once_in_a_While",
+    "image": require('./Halte_Once_in_a_While.png')
+  },
+  {
+    "imageId": "j2C_charge",
+    "image": require('./j2C_charge.png')
+  },
+  {
+    "imageId": "jC_charge",
+    "image": require('./jC_charge.png')
+  },
+  {
+    "imageId": "2C",
+    "image": require('./2C.png')
+  },
+  {
+    "imageId": "2A",
+    "image": require('./2A.png')
+  },
+  {
+    "imageId": "jB",
+    "image": require('./jB.png')
+  },
+  {
+    "imageId": "66B",
+    "image": require('./66B.png')
+  },
+  {
+    "imageId": "Plumage_Dancing_in_the_Wind",
+    "image": require('./Plumage_Dancing_in_the_Wind.png')
+  },
+  {
+    "imageId": "Breezs_Liberte",
+    "image": require('./Breezs_Liberte.png')
+  },
+  {
+    "imageId": "5C",
+    "image": require('./5C.png')
+  },
+  {
+    "imageId": "Chasing_le_Reve",
+    "image": require('./Chasing_le_Reve.png')
+  },
+  {
+    "imageId": "5C_charge",
+    "image": require('./5C_charge.png')
+  },
+  {
+    "imageId": "Lumiere_of_the_Dawn",
+    "image": require('./Lumiere_of_the_Dawn.png')
+  },
+  {
+    "imageId": "Agnes_Invitation",
+    "image": require('./Agnes_Invitation.png')
+  },
+  {
+    "imageId": "5A",
+    "image": require('./5A.png')
+  },
+  {
+    "imageId": "Throw",
+    "image": require('./Throw.png')
+  },
+  {
+    "imageId": "Let_the_Fleur_Carry_your_Feelings",
+    "image": require('./Let_the_Fleur_Carry_your_Feelings.png')
+  },
+  {
+    "imageId": "5B",
+    "image": require('./5B.png')
+  },
+  {
+    "imageId": "Advancer_With_You",
```

Comment:
[This move](http://wiki.mizuumi.net/w/Under_Night_In-Birth/UNIST/Nanase#Special_Moves) is called `Avancer With You`.